src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CLIRunner.java

```diff
@@ -191,6 +195,15 @@ private void validateConfigMode() {
         }
     }
 
+    private void validatePrConfigMode(CommandMode.PrConfigView comandMode) {
+        if (StringUtils.isNullOrEmpty(comandMode.getAppName()) || StringUtils.isNullOrEmpty(comandMode.getLatestCommit()) ||
+                StringUtils.isNullOrEmpty(comandMode.getPullRequestNumber()) || StringUtils.isNullOrEmpty(comandMode.getRepoUrl())) {
```

Comment:
You are not checking for target branch. Is that intentional?

Reply:
Done